src/components/UserProfile/AboutMe.js

```diff
@@ -43,6 +54,27 @@ export default compose(
   withState('isEditingBackgroundInfo', 'setEditingBackgroundInfo', false),
   withState('focusedTextArea', 'setFocusedTextArea', 'myBio'),
   withState('editingResearchInterests', 'setEditingResearchInterests', false),
+  withHandlers({
+    isEditingInfo: ({ setEditingBackgroundInfo, setEditingResearchInterests }) => ({
```

Comment:
here I think I'd prefer to just see two handlers, since they're basically two separate actions anyway. something like:
```
const trackProfileInteraction = ({ action, value }) => 
  trackUserInteraction({
    category: `User: Profile`,
    action: `Edit: ${action}`,
    label: value ? `open` : `close`
  });
...
withHandlers({
  handleEditingBackgroundInfo: ({ setEditingBackgroundInfo }) => ({
    type,
    value,
  }) => {
    setEditingBackgroundInfo(value);
    trackProfileInteraction({ action: 'Background Info', value });
  },
  // ... (repeat for research interests) 
})
```